spec/features/user_favorites_and_unfavorites_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+feature "User favorites and unfavorites" do
+# As a signed-in user
+# I want to favorite and unfavorite other users' profiles.
+#
+#
+# Acceptance Criteria
+# [ ] If I favorite another user's profile, the favorite count increments and I see a link to "Undo".
+# [ ] If I unfavorite another user's profile, the favorite count decrements and I see a link to "Favorite".
```

Comment:
Line is too long. [107/80]<br>Incorrect indentation detected (column 0 instead of 2).